### PR TITLE
Canvas: Improve placement when adding an element via context menu

### LINKED ISSUE #60190: Canvas: Improve placement when adding an element via context menu
When adding a new element via the context menu, it probably would make more sense for it to be initialized in the area of the context menu vs. the normal top / left initialization.

https://user-images.githubusercontent.com/22381771/207120232-2d3ea400-ac1b-41c8-b3dd-66afd7e213c6.mov

### PR DESCRIPTION
**Pull request by adela-almasan**
https://user-images.githubusercontent.com/88068998/210875856-04b7a1eb-f8af-4620-8df6-9ba443275976.mov

Fixes #60190

### CODE REVIEW

File: public/app/plugins/panel/canvas/CanvasContextMenu.tsx [hunk 1]
```diff
@@ -108,13 +109,23 @@ export const CanvasContextMenu = ({ scene, panel }: Props) => {
       const submenuItems: Array<
         React.ReactElement<MenuItemProps<unknown>, string | React.JSXElementConstructor<unknown>>
       > = [];
+
+      const onClickItem = (option: SelectableValue<string>) => {
+        let offsetY = 0;
+        if (scene.div) {
+          const sceneContainerDimensions = scene.div.getBoundingClientRect();
+          offsetY = anchorPoint.y - sceneContainerDimensions.top;
+        }
```
Comment (nmarrs) [inline public/app/plugins/panel/canvas/CanvasContextMenu.tsx:118]: Great job on this! Added a small change to offset y by scene container so added element aligns better with anchor point 


https://user-images.githubusercontent.com/22381771/210879864-fcf20fef-9a14-47cd-baca-c242735eba17.mov

**Review by nmarrs — APPROVED** [review]
⭐️

File: public/app/plugins/panel/canvas/CanvasContextMenu.tsx (hunk 1 above)
Comment (adela-almasan) [reply public/app/plugins/panel/canvas/CanvasContextMenu.tsx:120]: Thanks for the update 😁 I chose not to offset it as to me it looks a bit off, while using just the anchor point the item gets displayed right where the item was chosen from the list. Personal preference ofc, but it seems more natural to me 😁

Comment (nmarrs) [reply public/app/plugins/panel/canvas/CanvasContextMenu.tsx:120]: Ah I see your point there. My thinking was if we have it offset from anchor point there is potential for user to add an element that is off screen / vs. we can ensure added element is always on screen if we originate it at the anchor point for the context menu

**Pull request merged**